marc_to_solr/translation_maps/holding_library.rb

```diff
@@ -13,6 +13,7 @@
   "engineer$pt" => "ReCAP",
   "arch$pw" => "ReCAP",
   "marquand$pz" => "ReCAP",
+  "marquand$pv" => "ReCAP",
   "mendel$qk" => "ReCAP",
```

Comment:
@kevinreiss @mzelesky I also updated the holding_library since it is a remote recap. Is this correct?

Reply:
Yes. This is correct.